cluster/gce/util.sh

```diff
@@ -636,37 +636,45 @@ function write-windows-node-env {
   build-kubelet-config false "windows" "${KUBE_TEMP}/windows-node-kubelet-config.yaml"
 }
 
-function build-linux-node-labels {
+# Outputs node labels common to both Linux and Windows nodes.
+function build-common-node-labels {
   local master=$1
   local node_labels=""
   if [[ "${KUBE_PROXY_DAEMONSET:-}" == "true" && "${master}" != "true" ]]; then
     # Add kube-proxy daemonset label to node to avoid situation during cluster
-    # upgrade/downgrade when there are two instances of kube-proxy running on a node.
+    # upgrade/downgrade when there are two instances of kube-proxy running on a
+    # node.
     node_labels="beta.kubernetes.io/kube-proxy-ds-ready=true"
```

Comment:
Hmm...we actually don't want this label on windows nodes since we don't run kube-proxy as DaemonSets. I removed it in my PR.

Reply:
Whoops, I got the change to fluentd-ds-ready from your PR but missed kube-proxy-ds-ready. Fixed.